apps/judicial-system/api/src/app/modules/user/dto/createUser.input.ts

```diff
@@ -0,0 +1,40 @@
+import { Allow } from 'class-validator'
+
+import { Field, InputType } from '@nestjs/graphql'
+
+import { CreateUser, UserRole } from '@island.is/judicial-system/types'
+
+@InputType()
+export class CreateUserInput implements CreateUser {
+  @Allow()
+  @Field()
+  readonly nationalId!: string
+
+  @Allow()
+  @Field()
+  readonly name!: string
+
+  @Allow()
+  @Field()
+  readonly title!: string
+
+  @Allow()
+  @Field()
+  readonly mobileNumber!: string
+
+  @Allow()
+  @Field()
+  readonly email!: string
+
+  @Allow()
+  @Field(() => String)
```

Comment:
Is this correct? Why is this needed here but not in the other fields?

Reply:
The reason is the enum type UserRole. The GraphQl schema generator cannot resolve it without help.